### PR TITLE
Fix some dead links

### PR DESCRIPTION
**Pull request by wurstsalat3000**
This fixes some 404s found by running [linkchecker](https://linkchecker.github.io/linkchecker/) against xmpp.org: `linkchecker https://xmpp.org --ignore-url=/extensions --file-output=csv`

* Fixes various 404s in content files
* Furthermore, this fixes a linking issue with externals URLs in the navigation sidebar

### CODE REVIEW

File: content/blog/2010-xmpp-interop-event-update.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ We also had quite a few client software teams who participated:
 -   [SAFEchat](http://www.boldonjames.com/)
 -   [Swift](http://swift.im/)
 -   [Gajim](http://gajim.org/)
--   [Telepathy-Gabble](http://)
+-   Telepathy-Gabble
```
Comment (Zash) [inline content/blog/2010-xmpp-interop-event-update.md:25]: `https://telepathy.freedesktop.org/components/telepathy-gabble/` maybe?

Comment (wurstsalat3000) [reply content/blog/2010-xmpp-interop-event-update.md]: Thanks!

File: content/blog/freedom.md [hunk 2]
```diff
@@ -5,14 +5,14 @@ author: stpeter
...
 
 Over the years that sense of freedom has broadened and deepened. Consider:
 
 -   Our community has centered on Internet protocols (not a given open-source codebase) in part to give developers the freedom to use whatever license they want: free source, open source, freeware, shareware, commercial, or any combination thereof.
 -   Our protocols are as free as the air, which means anyone may implement them without asking permission or worrying about patents.
 -   Our protocols are pure XML, which means developers may extend the protocols for their own custom functionality. While we standardize [XMPP extensions](https://xmpp.org/extensions) that are of more general interest, developers have the freedom to make their own extensions and not standardize them through the XSF if they don't want to.
--   As [previously mentioned](http://blog.xmpp.org/?p=9), all of our discussion venues and processes are completely open and transparent, giving any individual the freedom to participate.
+-   As previously mentioned, all of our discussion venues and processes are completely open and transparent, giving any individual the freedom to participate.
```
Comment (Zash) [inline content/blog/freedom.md:15]: Looks like that https://web.archive.org/web/20070218205953/http://blog.xmpp.org/?p=9 would be this page https://xmpp.org/2007/02/be-open/ now?

Comment (Echolon) [reply content/blog/freedom.md:15]: But this is good, right? The new rendering?

File: content/blog/testing-1-2-3.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ author: stpeter
 categories: ["Miscellaneous"]
 ---
 
-One of the [roadmap](https://xmpp.org/xsf/roadmap.shtml) items we discussed at the [XMPP Summit](http://blog.xmpp.org/?p=12) in late February was the need to certify that XMPP clients and servers comply with our protocol specifications. 
+One of the [roadmap](/about/xsf/roadmap) items we discussed at the XMPP Summit in late February was the need to certify that XMPP clients and servers comply with our protocol specifications.
```
Comment (Zash) [inline content/blog/testing-1-2-3.md:8]: Another WordPress `?p=(number)` link, looks like the current URL is https://xmpp.org/2007/02/brussels-report/

File: content/blog/last-call-sasl-external.md [hunk 4]
```diff
@@ -7,7 +7,7 @@ categories: ["Miscellaneous"]
 
 Today we [issued](https://mail.jabber.org/pipermail/standards/2007-January/013676.html) a Last Call regarding [XEP-0178: Best Practices for Use of SASL EXTERNAL](https://xmpp.org/extensions/xep-0178.html). This specification is part of our [continuing effort](http://blog.xmpp.org/?p=4) to advance the XMPP RFCs to Draft Standard within the IETF.
```
Comment (Zash) [inline content/blog/last-call-sasl-external.md:8]: [`?p=4`](https://web.archive.org/web/20070629021227/http://blog.xmpp.org/?p=4) looks like https://xmpp.org/2007/01/continuing-the-conversation-at-the-ietf/

**Review by Zash — CHANGES_REQUESTED** [review]
Spotted some more wordpress links that could be updated to current locations by looking at the internet archive. `grep -F "blog.xmpp.org/?p=" -R content/` might be a good thing to scan.

File: content/blog/testing-1-2-3.md (hunk 3 above)
Comment (wurstsalat3000) [reply content/blog/testing-1-2-3.md]: Thanks! Found some more and fixed them as well (thank you, internet archive).

**Comment by wurstsalat3000** [comment]
@Zash you requested changes, but I `push -f`'d, sorry. Could you point me to where you want changes again, please?

**Review by Zash — APPROVED** [review]
Awesome!

**Comment by Zash** [comment]
I think you got all the wordpress links now :)

**Pull request merged**